package.json

```diff
@@ -1,365 +1,366 @@
-    "compile:views": "webpack --mode development",
-    "watch": "npm-run-all -p watch:*",
-    "watch:extension": "tsc --watch",
-    "watch:views": "webpack --watch --mode development",
-    "pretest": "npm run compile",
-    "test": "npm-run-all test:*",
-    "test:extension-tests": "node ./out/test/runTest.js",
-    "test:ts": "jest",
-    "test:api-tests": "pytest src",
-    "lint": "npm-run-all lint:*",
-    "lint:ts": "tslint -c tslint.json src/**/*.{ts,tsx}",
-    "lint:python": "pylint src",
-    "format": "npm-run-all format:*",
-    "format:ts": "prettier --config .prettierrc.yaml --write src/**/*.{css,ts,tsx}",
-    "format:python": "black src",
-    "check": "npm-run-all check:*",
-    "check:ts": "prettier --config .prettierrc.yaml --check src/**/*.{css,ts,tsx}",
-    "check:python": "black src --check",
-    "package": "vsce package"
-  },
-  "devDependencies": {
-    "@types/glob": "^7.1.1",
-    "@types/node": "^10.12.21",
-    "@types/react": "16.8.6",
-    "@types/react-dom": "16.8.4",
-    "@types/vscode": "^1.34.0",
-    "css-loader": "^1.0.0",
-    "del": "^4.0.0",
-    "event-stream": "^4.0.1",
-    "gulp": "^4.0.2",
-    "gulp-cli": "^2.1.0",
-    "gulp-filter": "^5.1.0",
-    "gulp-sourcemaps": "^2.6.5",
-    "gulp-typescript": "^5.0.1",
-    "less": "^3.7.0",
-    "less-loader": "^4.1.0",
-    "mocha": "^6.1.4",
-    "npm-run-all": "^4.1.3",
-    "prettier": "^1.19.1",
-    "react-scripts": "^3.3.1",
-    "style-loader": "^0.21.0",
-    "ts-import-plugin": "^1.5.4",
-    "ts-loader": "^4.4.2",
-    "tslint": "^5.12.1",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-microsoft-contrib": "^6.1.0",
-    "tslint-react": "^3.6.0",
-    "tslint-react-hooks": "^2.0.0",
-    "typescript": "^3.3.1",
-    "typescript-react-intl": "^0.4.0",
-    "version-from-git": "^1.1.1",
-    "vsce": "^1.47.0",
-    "vscode-nls-dev": "^3.3.1",
-    "vscode-test": "^1.0.0",
-    "webpack": "^4.15.1",
-    "webpack-cli": "^3.0.8"
-  },
-  "dependencies": {
-    "@babel/preset-typescript": "^7.8.3",
-    "@testing-library/jest-dom": "^5.0.2",
-    "@testing-library/react": "^9.4.0",
-    "@types/jest": "^24.9.0",
-    "@types/open": "^6.1.0",
-    "@types/react-test-renderer": "^16.9.0",
-    "@types/socket.io": "^2.1.2",
-    "babel-jest": "^25.1.0",
-    "compare-versions": "^3.5.1",
-    "eventemitter2": "^5.0.1",
-    "glob": "^7.1.4",
-    "jest": "^25.1.0",
-    "jest-transform-css": "^2.0.0",
-    "office-ui-fabric-react": "^7.85.0",
-    "open": "^6.4.0",
-    "os": "^0.1.1",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-intl": "^3.1.9",
-    "react-test-renderer": "^16.9.0",
-    "socket.io": "^2.2.0",
-    "svg-inline-react": "^3.1.0",
-    "ts-jest": "^25.0.0",
-    "util": "^0.12.1",
-    "vscode-extension-telemetry": "^0.1.1",
-    "vscode-nls": "^4.1.0"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "extensionDependencies": [
-    "ms-python.python"
-  ]
 }
```

Comment:
Any reason for this re ordering?

Reply:
I accidentally re-ordered when resolving merge conflicts :( is it ok if we just leave it like this?

Reply:
yeah sure no issues